release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -764,6 +782,14 @@ In {product-title} 4.7, the Insights Operator collects the following additional
 
 With this additional information, Red Hat can provide improved remediation steps in {cloud-redhat-com}.
 
+[id="ocp-4-7-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-7-auth-tp-aws-sts"]
+==== Running {product-title} using AWS Security Token Service (STS) for credentials (Technology Preview)
+You can now configure the Cloud Credential Operator (CCO) to use the Amazon Web Services Security Token Service (AWS STS). When the CCO is configured to use STS, it assigns components IAM roles that provide short-term, limited-privilege security credentials.
+
+For more information, see xref:../ authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc[Support for Amazon Web Services Secure Token Service (AWS STS)].
```

Comment:
I don't see this file merged yet. though I could be missing something.

xref:../authentication   (extra space in the path before authentication)

The path needs a section ID added after the .adoc#